os2forms_failed_jobs.install

```diff
@@ -47,6 +49,13 @@ function os2forms_failed_jobs_schema(): array {
   return $schema;
 }
 
+/**
+ * Add existing queue elements to new table.
+ */
+function os2forms_failed_jobs_install(): void {
+  \Drupal::getContainer()->get(Helper::class)->handleImport();
```

Comment:
Use `Drupal::service(Helper::class)` to get the service as we do it throughout the OS2Forms project.

Reply:
Fixed